module/extension/Director/src/Director.hpp

```diff
@@ -29,6 +31,10 @@
 namespace module::extension {
 
     class Director : public NUClear::Reactor {
+    public:
+        /// A task list is just a list of tasks
+        using TaskList = std::vector<std::shared_ptr<const ::extension::behaviour::commands::DirectorTask>>;
```

Comment:
```suggestion
        using TaskList = std::vector<std::shared_ptr<const ::extension::behaviour::commands::DirectorTask>>;
        /// Add the reason why a pack of tasks is different to a list of tasks.
        using TaskPack = std::vector<std::shared_ptr<const ::extension::behaviour::commands::DirectorTask>>;
```
If you want to separate `TaskList` and `TaskPack` either do it thoroughly to avoid confusion.